js/adapt-assessmentArticleModel.js

```diff
@@ -27,7 +27,9 @@ const AssessmentModel = {
   _getCurrentQuestionComponents() {
     return this.findDescendantModels('block')
       .filter(block => block.get('_isAvailable'))
-      .reduce((questions, block) => questions.concat(block.findDescendantModels('question')), []);
+      .reduce((questions, block) => questions.concat(block.findDescendantModels('question')
+        .filter(value => !questions.includes(value))),
```

Comment:
```js
this.findDescendantModels('block')
```
This will always return unique blocks so I dont thing the question uniqueness check is required